### PR TITLE
feat: refactor agreement modal and create folders to classify components

### PR DESCRIPTION
**Pull request by leo220yuyaodog**

### CODE REVIEW

File: web/src/table/SignupTable.js [hunk 1]
```diff
@@ -172,25 +172,25 @@ class SignupTable extends React.Component {
...
+              {id: "No verification", name: i18next.t("general:No verification")},
             ];
           } else if (record.name === "Agreement") {
             options = [
-              {id: "None", name: "None"},
-              {id: "Signin", name: "Signin"},
-              {id: "Signin (Default True)", name: "Signin (Default True)"},
+              {id: "None", name: i18next.t("general:Only signup")},
+              {id: "Signin", name: i18next.t("general:Signin")},
+              {id: "Default True", name: i18next.t("general:Signin (Default True)")},
```
Comment (hsluoyz) [inline web/src/table/SignupTable.js:193]: id should be `"Signin (Default True)"`

File: web/src/locales/zh/data.json [hunk 2]
```diff
@@ -621,6 +632,7 @@
     "The input is not valid Email!": "您输入的电子邮箱格式有误！",
     "The input is not valid Phone!": "您输入的手机号格式有误！",
     "Username": "用户名",
+    "Username - Tooltip": "用户名",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json:635]: Where is it?

File: web/src/locales/zh/data.json [hunk 3]
```diff
@@ -305,6 +314,8 @@
     "Logging out...": "正在退出登录...",
     "No account?": "没有账号？",
     "Or sign in with another account": "或者，登录其他账号",
+    "Password": "密码",
+    "Password - Tooltip": "密码",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json:318]: Where is it?

File: web/src/locales/ru/data.json [hunk 4]
```diff
@@ -200,8 +202,12 @@
     "Models": "Модели",
     "Name": "Имя",
     "Name - Tooltip": "Уникальный идентификатор на основе строки",
+    "No verification": "No verification",
+    "None": "None",
+    "Normal": "Normal",
```
Comment (hsluoyz) [inline web/src/locales/ru/data.json:207]: Put them to "application" instead of "general"

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/locales/zh/data.json (hunk 2 above)
Comment (leo220yuyaodog) [reply web/src/locales/zh/data.json:635]: > Where is it?
I'm also confused about it, it should has generated before.

image

File: web/src/ProviderEditPage.js [hunk 5]
```diff
@@ -130,6 +130,24 @@ class ProviderEditPage extends React.Component {
     }
   }
 
+  getProviderSubTypeOptions(type) {
+    if (type === "WeCom" || type === "Infoflow") {
+      return (
+        [
+          {id: "Internal", name: "Internal"},
```
Comment (hsluoyz) [inline web/src/ProviderEditPage.js:137]: i18n

File: web/src/ProviderEditPage.js [hunk 6]
```diff
@@ -130,6 +130,24 @@ class ProviderEditPage extends React.Component {
...
+          {id: "Third-party", name: "Third-party"},
+        ]
+      );
+    } else if (type === "Aliyun Captcha") {
+      return [
+        {id: "nc", name: "Sliding Validation"},
```
Comment (hsluoyz) [inline web/src/ProviderEditPage.js:143]: i18n

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**